rad-profile/src/cli.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright © 2021 The Radicle Link Contributors
+//
+// This file is part of radicle-link, distributed under the GPLv3 with Radicle
+// Linking Exception. For full terms see the included LICENSE file.
+
+pub mod args;
+pub mod main;
+
+pub use main::main;
```

Comment:
How does this interact with the `pub mod main;` above?

Reply:
I believe Rust knows what's up. I was playing around with a `src/main.rs` locally and I made this change to make it more sensible than `rad_exe::main::main()` :smile: